packages/kit/test/apps/basics/src/routes/before-navigate/preserve-search-params/+layout.svelte

```diff
@@ -0,0 +1,22 @@
+<script>
```

Comment:
The formatting seems a bit messed up in this file, can you run prettier in it?

Reply:
@ghostdevv good catch, actually not sure why it hadn't caught the double-quotes the first time! :')